frontend/src/app/core/services/tile.service.ts

```diff
@@ -51,4 +52,25 @@ export class TileService {
     convertTileSettingsToJson(tileSettings: object): string | undefined {
         return tileSettings ? JSON.stringify(tileSettings) : undefined;
     }
```

Comment:
It is better to put pure functions in a separate file, something like tiles.utils.ts. And make this function arrow so that it does not create its own scope.